### ISSUE TITLE
Order Incorrect for Match Videos page

### ISSUE THREAD

**Issue by Techno11** [lethosor/TBA-uploader#23]
See screenshot
![image](https://user-images.githubusercontent.com/15097541/193475299-67c82fdc-314b-4f0a-bc6e-ce21a5f87407.png)

Possibly do a selector for playoff/quals matches?

**Comment by lethosor** [comment]
Yeah, a selector is probably a good idea. It's a pretty crude sorting algorithm right now (a/b are just the match codes):
https://github.com/lethosor/TBA-uploader/blob/1c10f735733d77633b7a20c05cc22b0402488cee/web/src/components/App.vue#L2052
